protocol/metrics/rpcconsumerlogs.go

```diff
@@ -154,6 +154,18 @@ func (rpccl *RPCConsumerLogs) LogStartTransaction(name string) func() {
 	}
 }
 
+func (rpccl *RPCConsumerLogs) AddMetricForProcessingLatencyBeforeProvider(analytics *RelayMetrics, chainId string, apiInterface string) {
+	if analytics != nil && analytics.ProcessingTimestamp.Before(time.Now()) {
+		rpccl.consumerMetricsManager.SetRelayProcessingLatencyBeforeProvider(time.Since(analytics.ProcessingTimestamp), chainId, apiInterface)
```

Comment:
use a routine to set the value 
```suggestion
		go rpccl.consumerMetricsManager.SetRelayProcessingLatencyBeforeProvider(time.Since(analytics.ProcessingTimestamp), chainId, apiInterface)
```

Reply:
done. see https://github.com/lavanet/lava/commit/86b7f3173a1068378b2a1e327e9082bb37ef2a90